integration-tests/utils/Login.ts

```diff
@@ -30,6 +30,10 @@ export class Login {
   private static waitForApps() {
     Common.waitForLoad();
     Common.verifyPageTitle(pageTitles.overviewPage);
+    cy.get('[data-test="view-my-applications"]')
+      .should('be.visible')
+      .and('have.attr', 'aria-disabled')
+      .and('equal', 'false');
```

Comment:
if be.enabled isn't working then I am ok with this.
can we move 'false' to  `.and('have.attr', 'aria-disabled', 'false')` it should also work.